vue/vue.config.js

```diff
@@ -1,3 +1,8 @@
+process.env.VUE_APP_ROLLBAR_ENVIRONMENT = process.env.HEROKU_APP_NAME
+  || process.env.RAPID_SKILLS_ENVIRONMENT
+  || process.env.NODE_ENV
+  || 'production';
```

Comment:
@jhash can you change `'production'` to  `process.env.RAILS_ENV`. That is more exact.